docs/en/wallet-solution.rst

```diff
@@ -2,56 +2,83 @@
 
 .. _wallet-solution.rst:
 
-wallet-solution.rst
+Wallet Solution
 +++++++++++++++++++++++++++
 
-[What is it]
-    
-[What it is usefull for]
-    
-[Example]
-    
-General Properties
-------------------
-    
-[TODO]
-
+The Wallet Solution is a comprehensive product offered by the Wallet Provider to cater to the needs of users in managing their digital assets securely. Designed to provide a seamless user experience, this solution enables users to leverage the capabilities of the Wallet effectively. 
+  
     
 Requirements
 ------------
 
- - req 1
- - req 2
+ - Trustworthiness within the Wallet ecosystem: the Wallet Instance must establish trust and reliability within the Wallet ecosystem.
+ - Compliance with Provider specifications for obtaining PID and (Q)EAA: the Wallet Instance must adhere to the specifications set by Providers for obtaining Personal Identification (PID) and Qualified or non-qualified Electronic Address Authentication (Q)EAA.
+ - Support for Android and iOS operating systems: the Wallet Instance should be compatible and functional on both Android and iOS operating systems, as well as available on Play Store and App Store respectively.
+ - Verification of device ownership by the user: the Wallet Instance must provide a mechanism to verify the user's actual possession of the device.
 
 
-Attributes
-----------
+          
+Wallet Solution
+-----------------------------
+It comprises a mobile app and backend services that work together to deliver a holistic Wallet experience.
 
-[Table with parameters/attributes]
-    
-.. list-table:: 
-   :widths: 20 60
-   :header-rows: 1
+The mobile app serves as the primary interface for users, allowing them to access and interact with their digital assets conveniently; such digital assets are the Attestations, that include Personal Identification Data (PID¹), a set of data that can uniquely identify a natural or a legal person, along with other Qualified and non-qualified Electronic Attestations of Attributes (QEAAs and EAAs respectively, or (Q)EAAs in a more concise fashion¹).
+Once a user installs the mobile app on its device, we refer to such an installation as Wallet Instance for the user.
 
-   * - **Claim**
-     - **Description**
-   * - key
-     - value
+Supporting the mobile app, the backend services play a vital role in ensuring the security and reliability of the Wallet Solution. These services are responsible for issuing the wallet instance attestation, a cryptographic proof that verifies the authenticity and integrity of the Wallet Instance. Additionally, the backend services handle revocation requests, allowing users to securely revoke access or privileges associated with their Wallet instance when needed.
```

Comment:
we don't have a clear definition of backend services, can we just rely to the Wallet Provider?

Reply:
It may be a futile argument, but I think backend services and Wallet Provider address two different concepts:
* backend services is a software component, part of the Wallet Solution
* Wallet Provider is a role in the ecosystem, coming with their duties and responsibilities.

There may be overlaps (for example, the Wallet Provider federation to the trust chain is implemented by the backend services); however, using different words may help keep concepts separated from their implementations.

Reply:
We dont have the definitivo of wallet backend in the ARF

If this component Is required for the wallet solutions It should be defined in the architecture

Privacy considerations should be also added, which data this backend would know about the user?

Reply:
In https://github.com/italia/eidas-it-wallet-docs/pull/15/commits/b943f77495143879df5c50fc4891dc26fe271dfe I replaced backend services with Wallet Provider. In addition, I removed any reference to revocations from Users, as it's not in scope and we don't have enough relevant details.

Wallet Provider receives data from Wallet Instance only in the attestation flow, which is better explained in the relative section.